x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -44,6 +44,11 @@ public enum Cap {
          */
         FN_CBRT,
 
+        /**
+         * Support for function {@code GREATEST}. Done in #98630.
+         */
+        FN_GREATEST,
```

Comment:
Added this for a `bit_length` test case, where I use the new function inside another function (see new test case `bitLengthInsideOtherFunction` in `string.csv-spec`)

Reply:
Thanks for the explanation :) Adjusted it with [Remove FN_GREATEST capability and only use the most recent in bitLeng…](https://github.com/elastic/elasticsearch/pull/115792/commits/666198e7c410df6ea59c9ce8327ada5f0497d375)